src/app/group/add-edit-group/add-edit-group.component.html

```diff
@@ -5,9 +5,9 @@
     <div class="modal-header">
 
         <!-- if clicked on add/ edit  -->
-        <h1 class="modal-title fs-5" id="exampleModalLabel">{{formHeading}}</h1>
+        <h1 class="modal-title fs-5" id="Form-heading">{{formHeading}}</h1>
```

Comment:
Don't use capitalize word in #id selector Form-heading

Reply:
Resolved.